test/fields/collections/Lexical/e2e/main/e2e.spec.ts

```diff
@@ -1243,20 +1243,20 @@ describe('lexicalMain', () => {
 
     const relationshipInput = page.locator('.drawer__content .rs__input').first()
     await expect(relationshipInput).toBeVisible()
-    await page.getByRole('heading', { name: 'Lexical Fields' })
+    page.getByRole('heading', { name: 'Lexical Fields' })
     await relationshipInput.click()
-    const user = await page.getByRole('option', { name: 'User' })
+    const user = page.getByRole('option', { name: 'User' })
```

Comment:
This seems to be a bug on github or git again. All changes in this file before line 1394 are not from this PR but from one that happened 3 hours ago: https://github.com/payloadcms/payload/pull/11075

I don't understand why they are shown in the diff of this one